### PR TITLE
chore(prettier): Upgrade to 2.3.x

### PR DESCRIPTION
**Pull request by evanpurkhiser**
https://prettier.io/blog/2021/05/09/2.3.0.html

Mostly good stuff

### CODE REVIEW

File: static/app/components/charts/barChartZoom.tsx [hunk 1]
```diff
@@ -88,14 +88,8 @@ class BarChartZoom extends React.Component<Props> {
...
-      const {
-        buckets,
-        location,
-        paramStart,
-        paramEnd,
-        minZoomWidth,
-        onHistoryPush,
-      } = this.props;
+      const {buckets, location, paramStart, paramEnd, minZoomWidth, onHistoryPush} =
+        this.props;
```
Comment (priscilawebdev) [inline static/app/components/charts/barChartZoom.tsx:92]: not sure about having `this.props` alone in one line but :D

**Review by priscilawebdev — APPROVED** [review]
🛳🇮🇹

File: static/app/components/charts/barChartZoom.tsx (hunk 1 above)
Comment (scttcper) [reply static/app/components/charts/barChartZoom.tsx:92]: yeah this is a weird one

**Comment by armenzg** [comment]
There's a linting error introduced by this (not a blocking issue):
image

**Pull request merged**